### ISSUE TITLE
windows and mac

### ISSUE THREAD

**Issue by Reaper10** [ssj71/infamousPlugins#33]
Is there any planes to bring these lv2 to windows and mac?

**Comment by ssj71** [comment]
Yes. I am slowly working on that, but don't hold your breath, lately I have only had a few hours each week to work. Currently I am working on making the UI's cross platform. But yes, hopefully before the end of summer there will be support for them.

**Comment by AnClark** [comment]
I'm now working on cross-platform support: <https://github.com/AnClark/infamousPlugins>.

By now VST2/VST3/CLAP versions are available on Linux and Windows. However, NTK is not cross-platform, so currently no GUI on Windows.
